sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page/data.jsp

```diff
@@ -0,0 +1,36 @@
+<%--@elvariable id="hcaExperimentsCount" type="int"--%>
+<%--@elvariable id="humanExperimentsCount" type="int"--%>
+
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ page contentType="text/html;charset=UTF-8" %>
+
+<div id="data" class="callout" data-equalizer-watch>
+    <h3><span class="icon icon-functional" data-icon="A"></span> Data</h3>
+    <hr>
+
+    <p> Single cell Expression Atlas provides access to the analysis results of
+        <fmt:formatNumber pattern="#,##0" value="${hcaExperimentsCount}"/> data sets from the Human Cell Atlas
+        and <fmt:formatNumber pattern="#,##0" value="${humanExperimentsCount}"/> data sets in <i>Homo sapiens</i>
+        from other studies.
+    </p>
+    <p>
+        All studies have been re-analysed using our standardised pipeline
+        (<a href="https://github.com/nunofonseca/irap">iRAP</a>). Pipelines and web services are
+        being developed by the Gene Expression Team at EMBL-EBI, funded by the
```

Comment:
Judging from https://wellcome.ac.uk/about-us, it looks like that under the new Wellcome Trust branding “the” isn’t used:
<img width="313" alt="image" src="https://user-images.githubusercontent.com/4425744/48912646-14f1bc00-ee6e-11e8-83b6-d5b58426598d.png">

Reply:
Now Wellcome is like your new best friend: “Here’s Wellcome! Say hi to all our users!”.

Reply:
"Welcome, Wellcome!"